README.org

```diff
@@ -19,9 +25,30 @@ For manual installation, clone the repository and call:
 
 * Config
 
-Bind the minor mode command in dired:
+** Bind the minor mode command in dired
 
 #+BEGIN_SRC elisp
 (with-eval-after-load 'dired
   (define-key dired-mode-map ")" 'dired-git-info-mode))
 #+END_SRC
+
+** Don't hide normal Dired file info
+
+By default, toggling =dired-git-info-mode= also toggles the built-in =dired-hide-details-mode=, which hides file details such as ownership, permissions and size. This behaviour can be disabled by overriding =dgi-auto-hide-details-p=:
+
+#+BEGIN_SRC elisp
+(with-eval-after-load 'dired
+  (setq dgi-auto-hide-details-p nil))
+#+END_SRC
+
+** Enable automatically in every Dired buffer (if in Git repository)
+
+To enable =dired-git-info-mode= whenever you navigate to a Git repository, add the following snippet:
+#+BEGIN_SRC elisp
```

Comment:
This is a good idea, I will add a function for that could you replace that section with `(add-hook 'dired-after-readin-hook 'dired-git-info-auto-enable)`?

Reply:
Did so!